tutorials/plot_events_and_annotations.py

```diff
@@ -0,0 +1,134 @@
+"""
+.. _tut_events_and_annotation_objects:
```

Comment:
remove this label, we do not need it (it's in other examples for historical reasons and those should be removed, too)

we can use the auto-generated `sphx_glr_...` labels instead